PWGJE/EMCALJetTasks/AliAnalysisTaskRhoSparseSV.h

```diff
@@ -0,0 +1,35 @@
+#ifndef AliAnalysisTaskRhoSparseSV_H
+#define AliAnalysisTaskRhoSparseSV_H
+
+// $Id: AliAnalysisTaskRho.h 58408 2012-09-03 07:00:58Z loizides $
+// $Id$
+
+#include "AliAnalysisTaskRhoBase.h"
+
+ class AliAnalysisTaskRhoSparseSV : public AliAnalysisTaskRhoBase {

```

Comment:
Hi Markus,   the rho task  that i have submitted is important for our b-jet analysis. You are right that this is a copy of the old  rho sparse task, however this old rho sparse task does not exist any more.   In october it was changed by Elian who has changed completely the structure and logic of rho evaluation.    To be compatible with our old results we need the old rho sparse task functionality.

Filip

Reply:
Hi @fkrizek ,

It would be helpful to get some more information what is missing in the standard rho task so this can be modified in a way that it is suitable for the various analysis rather than back porting old code. Adding @efranzis, @raymondEhlers  and @jdmulligan for more suggestions to the discussion.

Cheers

Markus